src/libstd/lib.rs

```diff
@@ -336,6 +336,9 @@ extern crate libc;
 #[allow(unused_extern_crates)]
 extern crate unwind;
 
+#[macro_use]
+extern crate cfg_if;
```

Comment:
Note that this is here because there's usage of a bare `cfg_if!` macro invocation in `stdsimd`. That needs to be updated to `cfg_if::cfg_if!` before this can be removed, but that can hopefully happen in parallel with this.